tests/test_full_features.py

```diff
@@ -0,0 +1,34 @@
+import os
+import re
+import glob
+import json
+from click.testing import CliRunner
+from P6.__main__ import main
+
+# Path to the “full” example workbook (e.g. created under tests/data/full_example.xlsx)
+FULL_XLSX = os.path.join(
+    os.path.dirname(__file__), "data", "Sydney_Python_transformation.xlsx"
+)
+
+
+def test_full_features_parse_creates_all_blocks(decompressed_hpo):
```

Comment:
The test receives a path to a decompressed HPO JSON file. However, HPO toolkit will decompress `hp.json.gz` on the fly (it picks up the `.gz` suffix).

So, you do not need to do the decompression, unless you really want to do it from some other reason.